aws-analytics-pinpoint/src/main/java/com/amplifyframework/analytics/pinpoint/AWSPinpointAnalyticsPlugin.java

```diff
@@ -64,22 +68,30 @@
     private static final String USER_NAME = "name";
     private static final String USER_EMAIL = "email";
     private static final String USER_PLAN = "plan";
+    private static final String AUTH_DEPENDENCY_PLUGIN_KEY = "awsCognitoAuthPlugin";
 
     private final Application application;
     private AutoEventSubmitter autoEventSubmitter;
     private AnalyticsClient analyticsClient;
     private AutoSessionTracker autoSessionTracker;
     private TargetingClient targetingClient;
+    private AWSCredentialsProvider credentialsProviderOverride; // Currently used for integration testing purposes
 
     /**
      * Constructs a new AmazonPinpointAnalyticsPlugin.
```

Comment:
Ah! Can you fix this and make it a reference?

Reply:
Done